dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/ApacheHttpClientChannels.java

```diff
@@ -329,7 +329,8 @@ public static ClientBuilder clientBuilder() {
         // dropwizard: 30 seconds (see idleTimeout in the linked docs)
         // https://www.dropwizard.io/en/latest/manual/configuration.html#Connectors
         // wc: 60 seconds (internal)
-        private static final TimeValue CONNECTION_INACTIVITY_CHECK = TimeValue.ofSeconds(4);
+        private static final TimeValue CONNECTION_INACTIVITY_CHECK = TimeValue.ofMilliseconds(
```

Comment:
@carterkozak are you intending to vary this at particular services, or just in in one-off test harnesses?

Reply:
Goal is to iterate with timelock reverting this to the original 20 second value in order to determine if/how much we hurt performance with #887. Hopefully there's no meaningful change and we remove the property.